gst/droidcamsrc/gstdroidcamsrcphotography.h

```diff
@@ -71,7 +71,12 @@ typedef enum
   PROP_MAX_EXPOSURE_TIME,
   PROP_NOISE_REDUCTION,
   PROP_EXPOSURE_MODE,
-  PROP_SUPPORTED_EXPOSURE_MODES
+  PROP_SUPPORTED_EXPOSURE_MODES,
```

Comment:
Can these move up into the first section? This section is for those that override GstPhoto properties. It'd be good to have all of the extra GstDroid properties grouped together, so we can possibly upstream them later.

Reply:
Also think they should be named consistently with the properties they describe, and maybe even listed in the same order. ie:
  PROP_SUPPORTED_WB_MODES,
  PROP_SUPPORTED_COLOR_TONES,
  PROP_SUPPORTED_SCENE_MODES,
  PROP_SUPPORTED_FLASH_MODES,
  PROP_SUPPORTED_FOCUS_MODES
  PROP_SUPPORTED_ISO_SPEEDS,